app/javascript/react/config/config.js

```diff
@@ -14,6 +14,11 @@ export const links = [
     component: Settings,
     menu_item_id: 'menu_item_settings'
   },
+  {
+    path: 'settings/conversion_hosts',
+    component: Settings,
+    menu_item_id: 'menu_item_settings'
+  },
```

Comment:
Just making sure...  we don't need an entry here for `settings/general` because by default, entering the `settings` route opens the general tab?

Reply:
@michaelkro That's correct. There's no `/settings/general` route, `/settings` and `/settings/conversion_hosts` are the routes to each tab.